.circleci/config.yml

```diff
@@ -23,7 +23,9 @@ jobs:
         - go-cache-v1-{{ checksum "/tmp/year" }}-{{ checksum "/tmp/month" }}
         - go-cache-v1-{{ checksum "/tmp/year" }}
 
-    - run: ./architect build
+    - run:
+        name: Build using architect
+        command: ./architect build
```

Comment:
I don't think you need that anymore when you use `architect/go-test-legacy`. Ping @tfussell.

Reply:
I haven't had a chance to finish my work here https://github.com/giantswarm/architect-orb/pull/90. Without that, you can't save the binary built with `go-architect-legacy` so it's not usable for projects that build a binary. This is an oversight on my part since I was focused on testing it with `apiextensions`. I can try to finish it tomorrow otherwise we're going to have every project on go modules before anyone can use this job I made.